snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml

```diff
@@ -1,14 +1,30 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  labels:
```

Comment:
should all of these changed be reflected in the helm charts / yamls as well?

Reply:
In my point of view it's just relevant for the operator certification process for IBM. It's not required in the charts / yamls. However we can include it there as complementary information, I can't see any problem. For my ambiguous answer I don't have preference. :)